modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/OrganizationResourceImpl.java

```diff
@@ -643,6 +643,11 @@ public Organization putOrganizationByExternalReferenceCode(
 						externalReferenceCode, contextCompany.getCompanyId());
 
 		if (serviceBuilderOrganization == null) {
+			if (Validator.isNull(organization.getExternalReferenceCode())) {
+				organization.setExternalReferenceCode(
+					() -> externalReferenceCode);
```

Comment:
is this required by sf? I mean.. Is it not enough `organization.setExternalReferenceCode(externalReferenceCode);` ?